### PR TITLE
docs(crashlytics): Add import statements to examples

### PR DESCRIPTION
**Pull request by AlphaGit**
### Description

The examples do not make it clear how to use the native helper exception recorder. This PR includes the import statements both for iOS and Android examples.

### Related issues

N/A

### Release Summary

N/A

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- ~~My change includes tests;~~
  - ~~[ ] `e2e` tests added or updated in `packages/\*\*/e2e`~~
  - ~~[ ] `jest` tests added or updated in `packages/\*\*/__tests__`~~
- ~~[ ] I have updated TypeScript types that are affected by my change.~~
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

N/A

:fire:

### CODE REVIEW

**Comment by AlphaGit** [comment]
It seems my PR needs a bit of tweaking before it's mergeable. Don't worry, I'll come back to it.

File: docs/crashlytics/usage/index.md [hunk 1]
```diff
@@ -258,6 +258,9 @@ In case you need to log non-fatal (handled) exceptions on the native side (e.g f
 ### Android
 
 ```java
+import io.invertbase.firebase.crashlytics.ReactNativeFirebaseCrashlyticsNativeHelper;
```
Comment (mikehardy) [inline docs/crashlytics/usage/index.md:261]:
```suggestion
import io.invertase.firebase.crashlytics.ReactNativeFirebaseCrashlyticsNativeHelper;
```

**Review by mikehardy — APPROVED** [review]
I like this is a lot - thanks for taking the time to post this, it should help others

All it needs is the CLA signed and I can merge it in

**Comment by AlphaGit** [comment]
@mikehardy All set -- thank you! I thought there were some other checks failing here.

**Pull request merged**